dataclasses.py

```diff
@@ -602,3 +603,72 @@ def _astuple_inner(obj, tuple_factory):
                           for k, v in obj.items())
     else:
         return deepcopy(obj)
+
+
+def replace(obj, **changes):
+    """Return a new object replacing specified fields with new values. This
```

Comment:
What about PEP 257 docstring style? One line summary, blank line, then more text.

Reply:
I'm not sure where I copied this style from, but on my list of things to do is review all of the docstrings. So thanks for the reminder.